test/recipes/90-test_store.t

```diff
@@ -10,10 +10,22 @@ use File::Spec;
 use File::Copy;
 use MIME::Base64;
 use OpenSSL::Test qw(:DEFAULT srctop_file srctop_dir bldtop_file data_file);
+use OpenSSL::Test::Utils;
 
 my $test_name = "test_store";
 setup($test_name);
 
+# Programs built with mingw have init code that mangles arguments that look
```

Comment:
Well, commentary is as misleading as original commit message. Even more so, as you don't really build "with mingw", but rather "*for* mingw". And of course again, it's not [mingw] init code that mangles things, but MSYS run-time when launching Windows/mingw binaries.

Reply:
Ah, sorry, forgot that part.  Will fix